docs/source/notes/cuda.rst

```diff
@@ -468,7 +468,7 @@ also preserve :class:`torch.device` and :class:`torch.dtype` of a Tensor).
 Use pinned memory buffers
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. warning:
+.. warning::
 
     This is an advanced tip. You overuse of pinned memory can cause serious
```

Comment:
Could we fix the grammar of "You overuse of pinned memory can cause serious" while we're at it? I think this is supposed to be "Your overuse of pinned memory" but maybe we can just write "If you overuse pinned memory this can cause problems if you are low on RAM"